src/lua/serializer.c

```diff
@@ -76,12 +76,26 @@ static struct {
 	OPTION(LUA_TBOOLEAN, encode_use_tostring, 0),
 	OPTION(LUA_TBOOLEAN, encode_invalid_as_nil, 0),
 	OPTION(LUA_TBOOLEAN, encode_error_as_ext, 1),
+	OPTION(LUA_TBOOLEAN, encode_escape_forward_slash, 0),
 	OPTION(LUA_TBOOLEAN, decode_invalid_numbers, 1),
 	OPTION(LUA_TBOOLEAN, decode_save_metatables, 1),
 	OPTION(LUA_TNUMBER,  decode_max_depth, 128),
 	{ NULL, 0, 0, 0},
 };
 
+int
+serializer_set_option_default(const char *name, int value)
+{
+	assert(name != NULL);
+	for (int i = 0; OPTIONS[i].name != NULL; i++) {
+		if (!strcmp(OPTIONS[i].name, name)) {
+			OPTIONS[i].defvalue = value;
+			return 0;
+		}
+	}
+	return -1;
```

Comment:
May be it would be right to add `diag_set()` here instead of the assert that you use on this function failure.

Reply:
The function is very internal, I feel that assert fits just as well.

Reply:
Changed to `unreachable()`.